core/dbt/events/types.py

```diff
@@ -593,9 +593,11 @@ def code(self) -> str:
 
     def message(self) -> str:
         if self.key_path == "":
-            description = f"{self.violation} at top level in file `{self.file}`"
+            description = (
+                f"{self.violation} at top level in file `{self.file}` is possibly a deprecation"
```

Comment:
nit: might be worth adding some more context or just link to a github issue/discussion with more context

Reply:
Added more detail 🙂